src/jsMain/kotlin/team/karakum/showcase/material/utils/ShowcaseContainer.kt

```diff
@@ -0,0 +1,22 @@
+package team.karakum.showcase.material.utils
+
+import mui.material.Box
+import mui.system.BoxProps
+import mui.system.sx
+import react.FC
+import web.cssom.Display
+import web.cssom.JustifyContent
+import web.cssom.px
+
+val ShowcaseContainer = FC<BoxProps> { props ->
```

Comment:
Looks like wrapper on 50%.
Internals can be hidden as I see.

Reply:
what about such changes? Introduced an external interface that extends just the needed BoxOwnProps

Reply:
removed the component completely